src/lib.rs

```diff
@@ -168,6 +174,10 @@ to only do so it when done deliberately and for good reasons.
 #![deny(missing_debug_implementations)]
 #![deny(missing_docs)]
 #![no_std]
+#![cfg_attr(
+    feature = "allocator_api",
+    feature(allocator_api, nonnull_slice_from_raw_parts)
```

Comment:
Is the `nonnull_slice_from_raw_parts` feature necessary in order to implement the `Allocator` API? If not, I'd like to avoid using it, and only depend on `allocator_api`. This will expose the crate to less nightly breakage and make it that much easier to go stable when the time comes.

Reply:
I believe that would require changing the implementation of `try_alloc_layout_fast` and `try_alloc_layout_slow`. Which ultimately will have to happen once you drop the Box and Vec implementation, but... If you have any other suggestion, I'm all ears, since this is scary business :D

Reply:
I guess it is fine then, if the `allocator_api` feature stabilizes before `nonnull_slice_from_raw_parts` (I kinda doubt it) then we can cross this bridge at that time.